src/chapters/data-types/exercises.rst

```diff
@@ -0,0 +1,64 @@
+Exercises: Data Types
+=====================
+
+To get started, open up Visual Studio and create a new console program by clicking *New* and selecting *Console Application* under *.Net Core App*.
+
+For each part of the exercises, carefully note in code comments which part of the exercises you are working on. 
+When you are done with that particular section of the exercises, comment out your solution and start working on the next one!
+
```

Comment:
I think it'd be good practice to have the students create a data-types-exercises solution and each of these items as a separate project. what do you think? that way they can keep each of their answers to the exercises in a working state - and they'll get practice running different projects in one solution and dealing with the Main methods/ program.cs

Reply:
the other thing is, we should talk again about the csharp-web-dev-exercises. I get now reading this taht it makes sense to have the students create their own solution program instead of adding projects to a starter repo, but maybe there's a packaging option like in java? i'll look into it

Reply:
Done! Thanks!